docs/source/dataset_formats.mdx

```diff
@@ -180,6 +180,8 @@ preference_example = {"prompt": "The sky is", "chosen": " blue.", "rejected": "
 preference_example = {"chosen": "The sky is blue.", "rejected": "The sky is green."}
 ```
 
+Some preference datasets can be found with [the tag `dpo` on Hugging Face Hub](https://huggingface.co/datasets?other=dpo). You can also explore the [librarian-bots' DPO Collections](https://huggingface.co/collections/librarian-bots/direct-preference-optimization-datasets-66964b12835f46289b6ef2fc) to identify preference datasets.
```

Comment:
Not for this PR, but WDYT about putting these Hub filters as a badge at the top of each trainer doc, similar to how `transformers` does it for architectures: https://huggingface.co/docs/transformers/model_doc/bert

![Screenshot 2024-10-01 at 17 31 20](https://github.com/user-attachments/assets/da452bde-3257-4389-88ea-2e1cfabddf4a)

You can generate these badges with: https://shields.io

Reply:
#2149